libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -686,6 +690,24 @@ void PbftManager::loopBackFinishState_() {
   current_step_clock_initial_datetime_ = std::chrono::system_clock::now();
 }
 
+void PbftManager::broadcastVotes(bool rebroadcast) {
+  auto [round, period] = getPbftRoundAndPeriod();
+  auto net = network_.lock();
+  if (net) {
+    if (auto soft_voted_block_data = getTwoTPlusOneSoftVotedBlockData(period, round);
+        soft_voted_block_data.has_value()) {
+      net->getSpecificHandler<network::tarcap::VotePacketHandler>()->onNewPbftVotes(
+          std::move(soft_voted_block_data->soft_votes_), rebroadcast);
+      vote_mgr_->sendRewardVotes(getLastPbftBlockHash(), rebroadcast);
+    }
+
+    LOG(log_dg_) << "Node " << node_addr_ << " broadcast next votes for previous round. In period " << period
```

Comment:
I guess this part `"Node " << node_addr_ << ` is pretty much useless

Reply:
Removed that part